.github/workflows/auto_pr_review.yaml

```diff
@@ -17,6 +17,7 @@ jobs:
     - uses: actions/labeler@v5.0.0
       with:
         dot: true
+        sync-labels: true
```

Comment:
I believe this will remove manual labels, but maybe that's fine in this repo. would be cool if it's all auto